custom/icds_reports/views.py

```diff
@@ -2178,8 +2178,9 @@ class GovernanceAPIView(View):
 
     @staticmethod
     def get_state_id_from_state_site_code(state_code):
-        return AwcLocation.objects.get(aggregation_level=AggregationLevels.STATE,
-                                       state_site_code=state_code, state_is_test=0).state_id
+        return AwcLocation.objects.filter(aggregation_level=AggregationLevels.STATE,
+                                          state_site_code=state_code, state_is_test=0).values(
+            'state_id')[0]['state_id']
```

Comment:
you could use, `values_list('state_id', flat=True)`.  Then you wont need to do `['state_id']` it will just be a list of state_ids with one element